code/controllers/subsystem/ships.dm

```diff
@@ -297,6 +297,9 @@ SUBSYSTEM_DEF(ship)
 	for(var/datum/objective/ftl/killships/O in SSstarmap.ship_objectives)
 		if(S.faction == O.faction)
 			O.ships_killed++
+	for(var/datum/objective/ftl/hold_system/O_h in SSstarmap.ship_objectives)
+		if(SSstarmap.current_system == O_h.target_system && O_h.wave_active)
+			O_h.ships_remaining --
```

Comment:
why the space between the var and the decrement

Reply:
i mean if it works it works just wanna make sure

Reply:
Old habit I guess? commit up to remove it